scripts/theia-version.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Was this code copied from somewhere?

Reply:
Script from theia-go-extensions

Reply:
I would be happy to put the licence, but there is non in the theia-go-extensions

Reply:
Note: as per the Foundation workflow .pdf, a CQ is still required because the original code is not under Foundation control. It should be a formality to have it accepted though.